luigi/worker.py

```diff
@@ -410,11 +410,7 @@ def __init__(self, scheduler=None, worker_id=None, worker_processes=1, assistant
                 pass
 
         # Keep info about what tasks are running (could be in other processes)
-        if worker_processes == 1:
-            self._task_result_queue = DequeQueue()
```

Comment:
Interesting. Before this patch, was this just a premature optimization done before?